packages/ember/tests/routing/router_service_test/currenturl_lifecycle_test.js

```diff
@@ -241,17 +245,17 @@ moduleFor(
             ['parent.index - parent.index', 'parent.child model', '/'],
             ['parent.index - parent.index', 'parent.loading activate', '/'],
             [
-              'parent.index - parent.index',
+              'parent.index - parent.loading',
               'parent.child routeWillChange: parent.index - parent.loading',
               '/',
             ],
             [
-              'parent.index - parent.index',
+              'parent.index - parent.loading',
               'parent.loading routeWillChange: parent.index - parent.loading',
               '/',
             ],
             [
-              'parent.index - parent.index',
+              'parent.index - parent.loading',
```

Comment:
These tests changed during the development of this PR, so this just gets it to pass with the work we've done here.  Not sure if this it totally correct?  Since we've changed how the router service gets invoked, I think this might be okay.

Reply:
Hello, thank you for updating this PR.
I was checking this change while you were actually writing you comment. See just below :)